Issue #2913775 Drush 9 support

fgm opened the pull request:
https://www.drupal.org/node/2913775

fgm committed:
Issue #2913775 by fgm: Convert commands to Drush 9. Step 1: converted 'rqqi' command.

fgm committed:
Issue #2913775: refactored rqqi to be compatible with Drush 8 and 9 using a service.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/FGM/rabbitmq/inspections/d421b519-55f0-425e-96ef-4ed1750a1a05) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/FGM/rabbitmq/inspections/d421b519-55f0-425e-96ef-4ed1750a1a05) completed: **3 new issues, 10 updated code elements**

phcorp committed:
Migrate drush `rqwk` command

phcorp committed:
Migrate drush `rqtp` command

phcorp committed:
Migrate drush `rqtc` command

phcorp committed:
Remove worker service

pguillard committed:
Proposal to decrease the log level for some messages (#1)

* Proposal to decrease the log level for some messages

* acknowledged to deleted

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/FGM/rabbitmq/inspections/4c04f364-08fa-4abb-b93c-95f6816962be) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/FGM/rabbitmq/inspections/4c04f364-08fa-4abb-b93c-95f6816962be) completed: **5 new issues, 11 updated code elements**

fgm committed:
Support the newly required extra.drush.services section in composer.json. Reorder composer.json.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/FGM/rabbitmq/inspections/9517e63c-c744-458c-8cd0-fe9b7034abd1) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/FGM/rabbitmq/inspections/9517e63c-c744-458c-8cd0-fe9b7034abd1) completed: **5 new issues, 11 updated code elements**

fgm committed:
Use the new colon-separated naming convention for Drush/Console commands.

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/FGM/rabbitmq/inspections/939484d5-5c06-4013-86b9-308aa94e98f1) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/FGM/rabbitmq/inspections/939484d5-5c06-4013-86b9-308aa94e98f1) completed: **5 new issues, 11 updated code elements**

phcorp committed:
Fix use of deprectated/removed constant WATCHDOG_INFO

[info] Use of undefined constant WATCHDOG_INFO - assumed 'WATCHDOG_INFO' Consumer.php:181 [0.429999999999999993 sec, 9.97000000000000064 MB]

fgm committed:
Merge remote-tracking branch 'pierre/patch-1' into 2913775-drush9

scrutinizer-notifier commented:
A [new inspection](https://scrutinizer-ci.com/g/FGM/rabbitmq/inspections/46de1a9a-9a9a-4b59-8e3b-b785e5131362) was created.

scrutinizer-notifier commented:
The [inspection](https://scrutinizer-ci.com/g/FGM/rabbitmq/inspections/46de1a9a-9a9a-4b59-8e3b-b785e5131362) completed: **5 new issues, 11 updated code elements**
